src/errors.rs

```diff
@@ -166,9 +166,11 @@ pub enum Error {
     #[error("Failed to delete requested data")]
     FailedToDelete,
 
+    /// The node hasn't left the section, and was not marked for relocation during reward operations
+    #[error("Node is not being relocated")]
+    NodeWasNotRelocated,
+
     /// Node error that was not expected for message response
     #[error("There was an unexpected error at the node: '{0}'")]
-    UnexpectedNodeError(String), // /// NetworkData error.
-                                 // #[error("Network data error:: {0}")]
-                                 // NetworkData(#[from] sn_data_types::Error)
+    UnexpectedNodeError(String),
```

Comment:
I feel this is one we should get rid of soon.

Reply:
Yeh I've removed it from node but for ther mapping to sn_messaging errors. So in node we have: 

``` match error {
        Error::InvalidOperation => ErrorMessage::InvalidOperation,
        Error::InvalidOwners(key) => ErrorMessage::InvalidOwners(key),
        Error::InvalidSignedTransfer(dot) => ErrorMessage::InvalidSignature,
        Error::TransferAlreadyRegistered => ErrorMessage::TransactionIdExists,
        Error::NoSuchChunk => ErrorMessage::NoSuchData,
        Error::NotEnoughSpace => ErrorMessage::NotEnoughSpace,
        Error::BalanceExists => ErrorMessage::BalanceExists,
        Error::TempDirCreationFailed(_) => ErrorMessage::FailedToWriteFile,
        Error::DataExists => ErrorMessage::DataExists,
        Error::NetworkData(error) => convert_dt_error_to_error_message(error),
        error => ErrorMessage::UnexpectedNodeError(error.to_string()),
    }```
    
    And that error is basically to avoid having to have _all node errors_ as msgs. Because not all of them should escape `node` as messages.